scintellometry/io/filehandlers.py

```diff
@@ -290,7 +290,7 @@ def read(self, size):
         while(iz < size):
             self._seek(self.offset)
             block, already_read = divmod(self.offset, self.filesize)
```

Comment:
The fact that this is needed means that either `self.offset` or `self.filesize` is not integer, which should not happen...